docs/ReactVaporStore.ts

```diff
@@ -1,5 +1,5 @@
-import { createStore, Store } from 'redux';
+import * as Redux from 'redux';
```

Comment:
I'm pretty sure it would work the way it was with a simple npm install, but when linking the module it doesn't. I think it is best the leave it as is (`import * as Redux`).